build-info-extractor/src/main/java/org/jfrog/build/extractor/clientConfiguration/client/response/GetRepositoriesKeyResponse.java

```diff
@@ -1,32 +1,13 @@
 package org.jfrog.build.extractor.clientConfiguration.client.response;
 
-import java.util.List;
-import java.util.stream.Collectors;
-
 public class GetRepositoriesKeyResponse {
```

Comment:
`GetRepositoriesKeyResponse` -> `GetRepositoriesResponse`
Right now we are only looking for keys but, this can be used by other types of 'get repositories' services in the future.